app/Models/Booth.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Booth extends Model
+{
+    use HasFactory;
+
+    /**
+     * Establishes the connection
+     * A booth belongs to a team
+     *
+     * @return BelongsTo
+     */
+    public function team(): \Illuminate\Database\Eloquent\Relations\BelongsTo
```

Comment:
It is more readable if the `\Illuminate\Database\Eloquent\Relations\BelongsTo` qualifier is replaced by an `import` (PHPStorm shoud suggest this as well)